messenger_dump.sql

```diff
@@ -0,0 +1,179 @@
+-- MySQL dump 10.13  Distrib 5.6.24, for Win64 (x86_64)
```

Comment:
Not an issue, but take a look at this video https://www.youtube.com/watch?v=emgJtr9tIME
I know some guys that consider MySQL among dbs as PHP among programming languages :D